utils/supabase.ts

```
@@ -1,17 +1,17 @@
+import type { Database } from "@/types/supabase";
+import { SUPABASE_ANON_KEY, SUPABASE_URL } from "@env";
 import AsyncStorage from "@react-native-async-storage/async-storage";
 import { createClient, type Session } from "@supabase/supabase-js";
 import type * as ImagePicker from "expo-image-picker";
 import * as FileSystem from "expo-file-system";
 import { decode } from "base64-arraybuffer";
-
-import { SUPABASE_URL, SUPABASE_ANON_KEY } from "@env";
-import type { FriendResponse, RequestResponse } from "@/types/Friend.interface";
-import type { User } from "@/types/User.interface";
+import * as FileSystem from "expo-file-system";
```

Biome (1.9.4)

[error] Shouldn't redeclare 'FileSystem'. Consider to delete it or rename it.

'FileSystem' is defined here:


(lint/suspicious/noRedeclare)